example/docs/drop.cpp

```diff
@@ -0,0 +1,20 @@
+
+// Copyright (c) 2023 Jiri Nytra (jiri.nytra at gmail dot com)
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+
+#include <flux.hpp>
+
+#include <cassert>
+#include <vector>
+
+int main()
+{
+    std::vector vec{1, 2, 3, 4, 5};
+
+    auto dropped = flux::from(vec).drop(3);
+
+    assert(flux::count(dropped) == 2);
```

Comment:
Could use `size()` here rather than `count()` to show that the adapted sequence is `sized` when the input sequence is

```suggestion
    assert(flux::size(dropped) == 2);
```